### PR TITLE
Add a peribolos configuration and documentation on how to use it.

### PR DESCRIPTION
**Pull request by dlorenc**
This is the first step in several to use periboos for Tektoncd org management,
through Tekton Tasks and Triggers. The other work will be done in the plumbing repo.
See here for more info: https://github.com/tektoncd/community/issues/30

### CODE REVIEW

**Comment by dlorenc** [comment]
This configuration was generated by using peribolos -dump, then stripping the parts that correspond to things other than org settings and org membership.

File: org/org.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- houshengbo
+- hrishin
+- iancoffey
+- jessm12
+- jkutner
+- joseblas
+- jromero
+- khrm
+- kimsterv
+- knative-prow-robot
```
Comment (vdemeester) [inline org/org.yaml]: Yeah, I think `knative-prow-robot` is not needed anymore :stuck_out_tongue_closed_eyes:

**Comment by dlorenc** [comment]
> Is the org then "frozen" somehow to prevent this file from going out of sync?

I don't know how to actually "freeze" it, but I'll send a PSA in slack :)

File: org/org.yaml (hunk 1 above)
Comment (dlorenc) [reply org/org.yaml]: Makes sense. I'll send a PR to drop that robot after we get this setup. It can be our first test!

File: org/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Tektoncd Github Organization
+
+This directory contains the configuration of the Tektoncd Github organization.
+To add or remove members and admins, make changes to this file.
+
+This system uses [peribolos](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos)
+to manage the org configuration.
+
+After config updates are made, an administrator must manually sync the configurations for them to take effect.
```
Comment (bobcatfish) [inline org/README.md]: I assume the manual sync part is just temporary?

File: org/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Tektoncd Github Organization
```
Comment (bobcatfish) [inline org/README.md]: Thanks for adding a README here!! Can you add a link from the main README to this?

File: org/README.md (hunk 2 above)
Comment (dlorenc) [reply org/README.md]: Yeah, I just need to get both halves setup, which I can't do until this is in.

**Comment by vdemeester** [comment]
@dlorenc nice :tada: I guess we can manage the teams (and their members) in a follow-up ? :angel:

**Pull request merged**